.github/workflows/prepare_release.yml

```diff
@@ -0,0 +1,83 @@
+name: Prepare release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: New version tag
```

Comment:
it should be required

Reply:
and we should add that it must start with `v`

Reply:
I made it required, but it must not start in a v, at least not in python.